README.md

```diff
@@ -50,9 +52,16 @@ if you're on campus use the `ubcsecure` or `resnet` networks for best results.
 - CLion will not support auto-completion in your *.cpp* and *.h* files until you've added them to the CMake file
 
 ### Zed Configuration
-- Follow the instructions on [this github page](https://github.com/stereolabs/zed-ros-wrapper) (this package already contains `zed_ros_wrapper` as a submodule in `src/zed_ros_wrapper`)
+- Follow the instructions on [this github page](https://github.com/stereolabs/zed-ros-wrapper) This repository already contains `zed_ros_wrapper` as a ~~submodule~~ standalone package in `external_pkgs/zed_ros_wrapper`(As of 2/25/2018 submodules have been migrated to rosinstall)
 - Download ZED calibration file from the link indicated when you run `zed.launch` and place it in the folder `/usr/local/zed/settings/`
 
+### Rosinstall
+
+To manually install dependent packages using rosinstall, run `rosinstall INSTALL_PATH PRE-EXISTING_ROS_DIR CUSTOM_ROSINSTALL_FILE` 
+i.e on most systems, it will be `rosinstall /usr/share/ros /opt/ros/kinetic .rosinstall` 
+ 
+Rostinstall is addressed and run in `install_dependencies.sh`, should there be any need to run manual, rosinstall should be run after all steps from installation and setup has been completed. 
+
```

Comment:
I think it would be really helpful if we had an `External Packages` section of the README with instructions on how to install and use external packages, now that we've got this updated structure.